README.md

```diff
@@ -28,9 +28,35 @@ of your proposal once assigned, e.g. `assets/aex-1/image.png`.
 
 Everything beyond step 4 is governed by [AEX-1](https://github.com/aeternity/AEXs/blob/master/AEXS/aex-1.md).
 
+## Status terms
+
+Read [AEX-1](https://github.com/aeternity/AEXs/blob/master/AEXS/aex-1.md) for more details.
+
+* Draft - an AEX that is open for consideration and is undergoing rapid iteration and changes.
+* Review - an AEX that is done with its initial iteration and in review by a wide audience.
+* Last Call - In review by a wide audience - last call for accepting changes.
+* Review - an AEX that is done with its initial iteration and ready for review by a wide audience.
```

Comment:
Now `Review` is in there twice

Reply:
Good catch. It seams that was the cause of the error - the duplication was there in my initial commit, so I just copy-pasted 2 times Review.